spec/bill_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'bill'
+
+describe Bill do
+
+  # let(:menu) { double :menu, :menu_items[:pie] => 4.5, :menu_items[:cranberry_juice] => 2.50 }
```

Comment:
This is a small thing, but try not to leave commented out code in anything you push to github.